libsolidity/ast/Types.cpp

```diff
@@ -3717,9 +3717,13 @@ MemberList::MemberMap MagicType::nativeMembers(ContractDefinition const*) const
 				{"creationCode", TypeProvider::array(DataLocation::Memory)},
 				{"runtimeCode", TypeProvider::array(DataLocation::Memory)},
 				{"name", TypeProvider::stringMemory()},
+				{"interfaceId", TypeProvider::fixedBytes(4)},
```

Comment:
Didn't we say to go with `interfaceID` because that's what EIP-165 uses?

Reply:
Ah yes, you're right.

Reply:
I also reserve to right to jump back to `interfaceId` when we see all the examples. It does fit Solidity naming better, and the only reason against it is what EIP-165 uses.

Reply:
> Didn't we say to go with interfaceID

I'm sorry @aarlt, it seems based on the last update (16 days ago) of the initial comment on #7856, we've ended up with `interfaceId`.

@chriseth @leonardoalt you were active on that call on the topic, how do you remember?

Reply:
@axic i'm somehow not able to see the last update that your'e referring too.. for me `interfaceId` looks nicer.. I will change it back to `interfaceId`